rerun_cpp/src/rerun/datatypes/rotation3d_ext.cpp

```diff
@@ -22,6 +22,10 @@ namespace rerun {
 #define Rotation3DExt Rotation3D
 #endif
 
-        const Rotation3DExt Rotation3DExt::IDENTITY = Rotation3DExt(Quaternion::IDENTITY);
+        // TODO(andreas): This constant initialization does not work for unknown reasons!
+        //                On clang(-mac) this set the Rotation3D::IDENTITY to all zero instead of
+        //                the expected quaternion.
+        // Using the same code as a non-constant works fine.
+        // const Rotation3DExt Rotation3DExt::IDENTITY = Quaternion::IDENTITY;
```

Comment:
oO

Reply:
wth

Reply:
I poked it for quite a while but couldn't figure it out, so I gave up for now :(

Reply:
`Rotation` is a union type to I reviewed that again, but can't see what's wrong with it